frontend/src/components/StatusTags.vue

```diff
@@ -31,16 +38,29 @@ export default {
   props: {
     popperPlacement: {
       type: String
+    },
+    showStatusText: {
+      type: Boolean,
+      default: false
     }
   },
   mixins: [shootItem],
   computed: {
     filteredConditions () {
-      if (isEmpty(this.shootConditions)) {
-        return []
-      }
-      return filter(this.shootConditions, condition => !!condition.lastTransitionTime)
+      return filter(this.shootReadiness, condition => !!condition.lastTransitionTime)
+    },
+    errorCodeObjects () {
+      const allErrorCodes = errorCodesFromArray(this.filteredConditions)
+      return objectsFromErrorCodes(allErrorCodes)
     }
   }
 }
 </script>
+
+<style lang="scss" scoped>
+
+  .wrap {
```

Comment:
```suggestion
  .wrap-text {
```
We should move this `main.scss` because we already have there `nowrap` and the same `.wrap-text` in at least 8 other compoments

Reply:
done